src/preferences/dialog/dlgprefdeck.cpp

```diff
@@ -495,11 +513,31 @@ void DlgPrefDeck::slotRateRampingModeLinearButton(bool checked) {
     }
 }
 
+void DlgPrefDeck::slotTimeFormatChanged(double v) {
+    int i = static_cast<int>(v);
+    m_timeDisplayFormat = static_cast<TrackTime::DisplayFormat>(i);
+
+    m_pConfig->set(ConfigKey("[Controls]","TimeFormat"), ConfigValue(v));
+    comboBoxTimeFormat->setCurrentIndex(
+                comboBoxTimeFormat->findData(i));
+
+}
+
+void DlgPrefDeck::slotTimeFormatChanged(int index) {
```

Comment:
I think we should get rid of this. It is anyway confusing to have this function twice as int and double overload.